android/quest/src/test/java/org/smartregister/fhircore/quest/ui/pin/PinViewModelTest.kt

```diff
@@ -62,6 +62,8 @@ class PinViewModelTest : RobolectricTest() {
     every { secureSharedPreference.retrieveSessionUsername() } returns "demo"
     pinViewModel.setPinUiState(true, context)
     val pinUiState = pinViewModel.pinUiState.value
+    val expectedMessage = "VHT will use this PIN to login"
+    Assert.assertNotNull(expectedMessage, pinUiState.message)
```

Comment:
shouldn't this be assertEquals?

Reply:
@pld  I have revised it to AssertEquals. Users can now configure and set their PinLoginScreen message to that of their choice